lib/resolve-config.js

```diff
@@ -31,6 +29,7 @@ module.exports = (
       : 'https://gitlab.com');
 
   return {
+    pkgRoot,
```

Comment:
I understand you are mirroring the NPM plugin here, but it feels a bit displaced here. The concept of a package does not really exist for the GitLab plugin.
What about `assetBasePath` or `assetRootPath`?

Reply:
Also we some documentation please 🙇